cli/submit.py

```diff
@@ -38,19 +43,25 @@ def get_submission_id_and_upload_url(self, submission_dir):
         else:
             raise FileNotFoundError(f'Could not upload. No config file found for the submission in {submission_dir}.')
 
-    # TODO
-    def upload_submission(self, submission_dir, submission_id=None, submission_upload_url=None):
-        if not submission_id or not submission_upload_url:
+    def upload_submission(self, submission_dir, submission_upload_url=None):
+        if not submission_upload_url:
             submission_id, submission_upload_url = self.get_submission_id_and_upload_url(submission_dir)
-        pass
+        for f in self.vcf_files:
+            self.upload_file(submission_upload_url, f)
```

Comment:
Maybe should log/print something for the user, as these uploads could take a while?